src/Forms/Prism.Forms/PrismApplicationBase.cs

```diff
@@ -140,15 +149,28 @@ private INavigationService CreateNavigationService(object view)
         /// </summary>
         protected virtual void Initialize()
         {
-            ContainerLocator.SetContainerExtension(CreateContainerExtension);
-            _containerExtension = ContainerLocator.Current;
-            RegisterRequiredTypes(_containerExtension);
-            PlatformInitializer?.RegisterTypes(_containerExtension);
-            RegisterTypes(_containerExtension);
-            _containerExtension.FinalizeExtension();
+            if(ContainerLocator.Current is null)
+            {
+                _isApplicationFirstRun = false;
+            }
+
+            if(!_isApplicationFirstRun)
```

Comment:
if this is the first run shouldn't we be executing this block of code?

Reply:
yes I updated the name but not the logic ;)